uktrade/directory-api
# Add ability to publish / un-publish case studies

This is 1 of 3 PRs that add functionally to publish / unpublish business case studies to Great CMS. 

Because of the complexity of the ticket, all three are required for the functionality to work. 

PRs (MUST be merged in this order):
1) directory-api (https://github.com/uktrade/directory-api/pull/1165)
2) international-ui (https://github.com/uktrade/great-international-ui/pull/950)
3) great-cms (https://github.com/uktrade/great-cms/pull/2356)

TO TEST:
1) Log into great CMS, and navigate to business profile (/profile/business-profile/)
2) In your business profile, scroll down, and create a new case study.
3) When creating a new case study, ensure radio dropdown for PUBLISHED / UNPUBLISHED is available on the form. Select Published.
4) When created, ensure the case study is visible on the main website. 
5) Go back into your case study and click 'edit'
6) Change the case study to 'unpublished'
5) Ensure case study is no longer visible on your business' public profile. 

### Workflow

- [ X] Ticket exists in Jira https://uktrade.atlassian.net/browse/KLS-829
- [ X] Jira ticket has the correct status.
- [ X] A clear/description pull request messaged added.

### Reviewing help

- [X ] Explains how to test locally, including how to set up appropriate data

### Housekeeping

- [X ] Cleaned up old feature flags
- [X ] Upgraded any vulnerable dependencies.

### Merging
